### ISSUE TITLE
trainloader issues

### ISSUE THREAD

**Issue by YanHanANU** [bnulihaixia/Deep_metric#12]
Hey I saw you updated your work recently, that is so nice of you.

I got a question about trainloader here. As I posted before, when I ran your code on my own computer, the loss function is increasing. The problem may be the trainloader. I found that in each epoch, the train loader only load about 480 images which is strange. The number of iterations in each epoch is only 6. I was trying to fix it by myself but it looks a little bit difficult to fully understand your code. Do you have any suggestions for me to fix this problem?

Cheers
Candice

**Comment by YanHanANU** [comment]
Thanks for your reply.
The version of pytorch I am using is 0.2.30. If you say that 6 iterations is correct, I would say that the codes are running well on my computer. 
I am just a little bit curious why each epoch does not go through all the training set. That is a little bit weird to me.

**Comment by bnulihaixia** [comment]
@CandiceUIC  Have you reached the performance as me?

**Comment by CandiceUIC** [comment]
@bnulihaixia For the Binomial Loss, I did reached a performance like you posted. But for other losses, for example triplet, I did not. I think that might be the reason of parameter choosing. I am now working on it. 

Anyway, thank you so much for posting your work!

cheers
